CMakeLists.txt

```diff
@@ -684,6 +700,29 @@ IF(USE_HDF5)
       ENDIF()
     ENDIF()
 
+
+
+    ###
+    # If HDF5_VERSION is still empty, we have a problem.
+    # Error out.
+    ###
+    IF("${HDF5_VERSION}" STREQUAL "")
+      MESSAGE(FATAL_ERR "Unable to determine HDF5 version.  NetCDF requires at least version ${HDF5_VERSION_REQUIRED}. Please ensure that libhdf5 is installed and accessible.")
+    ENDIF()
+
+    ####
+    # Check to see if HDF5 library is 1.10.6 or greater.
+    # Used to control path name conversion
+    ####
+    IF(${HDF5_VERSION} VERSION_GREATER "1.10.5")
```

Comment:
The 1.8.22 version of HDF5 works fine for me in CI testing with netCDF.

Reply:
@edwardhartnett Do you mean 1.8.22 works for you in regards to the `HDF5_UTF8_PATHS` toggle in this block? Or do you mean in general? Taking a look back, the `HDF5_VERSION_REQUIRED` "minimum version gate" is set to `1.8.10`.  Our CI tests against `1.8.21`, I'll take a look at bumping that to `1.8.22`.  I've just noticed a path by which `HDF5_VERSION` might not be checked to see if it meets the required version, so I'll be pushing that change as well.